tests/ui/missing_inline_test_crate.rs

```diff
@@ -0,0 +1,12 @@
+//@compile-flags: --test
+//@check-pass
+#![warn(clippy::missing_inline_in_public_items)]
+#![crate_type = "dylib"]
+// When denying at the crate level, be sure to not get random warnings from the
+// injected intrinsics by the compiler.
+#![allow(dead_code)]
```

Comment:
Are these two attributes necessary? I'm pretty sure they're both implicitly set. At least for me it passes just fine without them